packages/MarkdownEditor-Core.package/MarkdownAttribute.class/instance/end..st

```diff
@@ -0,0 +1,4 @@
+accessing
+end: anInteger
```

Comment:
`startPosition` / `endPosition` like in Tokens? Or are they different here?

Reply:
Here it is basically the start position of the opening token and the end position of the closing token

Reply:
It was meant as request to rename both instance variables. But rereading it, it could read as a question. When they both more or less have the same meaning, I think that `startPosition` and `endPosition` would be less confusing